src/webapp/pages/settings/SettingsPage.tsx

```diff
@@ -1,359 +1,328 @@
-                        <NotificationListTable
-                            isLoading={isNotificationLoading}
-                            notifications={notifications}
-                            onEditNotification={onEditNotification}
-                            deleteNotifications={deleteNotifications}
-                            saveNotifications={saveNotifications}
-                            fetchNotifications={fetchNotifications}
+                        <Select
+                            label={i18n.t("Language to add translation")}
+                            items={languages}
+                            onChange={setSelectedLang}
+                            value={selectedLang}
                         />
-                    </>
-                )}
-
-                <CreateButton landings={landings} onNewNotification={onNewNotification} />
-            </Container>
-        </DhisLayout>
-    );
-};
-
-function buildSharingDescription(props: Maybe<{ users?: NamedRef[]; userGroups?: NamedRef[] }>) {
-    const { users, userGroups } = { users: [], userGroups: [], ...(props || {}) };
-    const usersCount = users?.length ?? 0;
-    const userGroupsCount = userGroups?.length ?? 0;
-
-    if (usersCount > 0 && userGroupsCount > 0) {
-        return i18n.t("Accessible to {{usersCount}} users and {{userGroupsCount}} user groups", {
-            usersCount,
-            userGroupsCount,
-        });
-    } else if (usersCount > 0) {
-        return i18n.t("Accessible to {{usersCount}} users", { usersCount });
-    } else if (userGroupsCount > 0) {
-        return i18n.t("Accessible to {{userGroupsCount}} user groups", { userGroupsCount });
-    } else {
-        return i18n.t("Only accessible to system administrators");
+                    </Container>
+                </ConfirmationDialog>
+            </React.Fragment>
+        );
     }
-}
+);
 
-const Title = styled.h3`
-    margin-top: 25px;
+const Container = styled.div`
+    display: flex;
+    flex-flow: column;
 `;
 
-const Group = styled(FormGroup)`
-    margin-bottom: 35px;
-    margin-left: 0;
+const Select = styled(Dropdown)`
+    margin: 10px;
 `;
 
-const Container = styled.div`
-    margin: 1.5rem;
-`;
+export interface ImportTranslationRef {
+    startImport: () => void;
+}
 
-export const SubContainer = styled.div`
-    margin-bottom: 2rem;
-`;
+export interface ImportTranslationDialogProps {
+    type: "action" | "landing-page";
+    onSave: (key: string | undefined, lang: string, terms: Record<string, string>) => void | Promise<void>;
+}
 
-const Header = styled(PageHeader)`
-    margin-top: 1rem;
-`;
+function buildActionList(actions: Action[], translate: TranslateMethod): DropdownItem[] {
+    return actions.map(({ id, name }) => ({ value: id, text: translate(name) }));
+}
 
-export const GridForm = styled.div`
-    display: grid;
-    grid-template-columns: 2fr 1fr;
-    gap: 20px;
-`;
+const languages = [
```

Comment:
How come we need this list of languages now? In any case, it probably should not be hardcoded. Do we want to support any language? If so, there are npm packages with the ISO codes. Or do we want to support the DHIS2 locales? In that case, we'd get the list from the instance in some repo.